app/src/components/Courses/CourseListCard.tsx

```diff
@@ -2,136 +2,65 @@
 import { Course } from "../../interfaces/Course"
 
 // Components
-import StarRating from "../general/StarRating";
+
 import { Icon } from '@mdi/react';
-import { mdiPencil, mdiAccount } from '@mdi/js';
-import { LastEdited } from "./LastEdited";
 
-// Helpers
-import categories from "../../helpers/courseCategories";
-import statuses from "../../helpers/courseStatuses";
 
-// Images
-import imageNotFoundImage from '../../assets/image-not-found.png';
-import { useEffect, useState } from "react";
-import StorageServices from "../../services/storage.services";
 
-// Rating
-import { getAverageRatingOfCC, getAverageRatingOfCourse } from '../../services/contentCreator.services';
+import { mdiStar } from "@mdi/js";
+// Helpers
+
 
 /**
  * Displays a course in a card format
  * 
  * @param {Course} course The course to be displayed
  * @returns HTML Element
  */
-export const CourseListCard = ({ course }: { course: Course }) => {
-  const [imageSrc, setImageSrc] = useState<string>('');
-  const [isLoading, setIsLoading] = useState<boolean>(true);
-  const maxTitleLength = 20;
-  //Only load the picture, when the picture is loaded
-  useEffect(() => {
-    const fetchImage = async () => {
-      try {
-        if(course.coverImg == "" || course.coverImg == undefined) {
-          throw new Error("coverImg is empty or undefined");
-        }
-        const fileSrc = await StorageServices.getMedia(course.coverImg);
-
-        setImageSrc(fileSrc);
-
-      } catch (error) {
-        setImageSrc(imageNotFoundImage);
-      } finally {
-        setIsLoading(false);
-      }
-    };
-    const fetchRating = async () => {
-      try {
-        const rating = await getAverageRatingOfCourse(course._id);
-      } catch (error) {
-        console.error(error);
-      }
-    }
-
-    fetchRating();
-    fetchImage();
-  }, [course.coverImg, course.title]);
+export const CourseListCard = ({ course}: { course: Course },) => {
   return (
-    <div className="overflow-hidden shadow rounded h-full w-full cursor-pointer m-auto hover:shadow-lg duration-200">
-      <label
-        onClick={()=>window.location.href = `/courses/manager/${course._id}/0`}
-        className="w-full block h-full relative group"
+    <tr
+    key={course._id}
+    className="border-b border-gray-300 bg-white text-base font-['Montserrat']"
+  >
+    <td>
+      <div className="flex items-center px-5 py-5">
+          <p
+            className="text-gray-900 whitespace-no-wrap"
```

Comment:
Couldn't the custom Tailwind CSS property `grayDark` have been used here instead of `text-gray-900`?